schema/ridl/_example/example1.ridl

```diff
@@ -57,8 +73,26 @@ service ExampleService
 
   - stream SendAndRecv(req: string) => stream (resp: string)
 
-  - Broadcast() => stream (resp: Notice)
+  - Broadcast(qty?: Balance) => stream (resp: Notice)
 
   -	VerifyUsers(seq: int32, header?: map<string,[]string>, ids: []uint64) => (code: bool, ids: []bool)
 	
   - MoreTest(n: uint64, stuff: []map<uint64,string>, etc: string) => (code?: bool)
+
+
+
+struct FlattenRequest
+  - name: string
+    + go.tag.db = name
+  - amount?: Balance
+    + go.tag.db = amount
+
+struct FlattenResponse
+  - id: uint64
+    + go.field.name = ID
+  - count: uint64
+    + json = counter
+
+service Another
+  - Flatten(FlattenRequest) => (FlattenResponse)
```

Comment:
this is a nice new feature we have too in RIDL in this branch :)  it is a short-hand .. take a close look and see what it does